src/commands/generate-cr.ts

```diff
@@ -74,6 +74,16 @@ export default class GenerateCr extends Command {
       multiple: true,
       description: 'Accepted tag types, comma separated values. Accepted values are ' + [...GenerateCr.tagTypeStrategies.keys()].join(", ")
     }),
+    overwriteTenants: Flags.boolean({
+      char: 'v',
+      description: 'Overwrite the bundle cr tenants list with the value passed in the tenants parameter',
+      dependsOn: ['tenants']
+    }),
+    forceOverwriteTenants: Flags.boolean({
```

Comment:
you could remove dependsOn in `force` and reuse it

Reply:
The force flag and forceOverwriteTenants do different things, It's better IMHO to have 2 different flags

Reply:
in pipeline you don't have standard input, so a force flag to disable the requests to std input is sufficient.
Here, it is like we do:

```
entando-bundle-cli generate-cr --A --disableStdInputForA --B --disableStdInputForB
```
instead of
```
entando-bundle-cli generate-cr --A --B --disableStdInput
```

the first case will introduce an additional flag for each flag that needs to disable std input but we needs just one flag for the command to disable the std input (force).

Reply:
Yes, you'right. I'm going to do this change

Reply:
Done